interpreter/cling/lib/Interpreter/IncrementalParser.h

```diff
@@ -68,9 +71,6 @@ namespace cling {
     // file ID of the memory buffer
     clang::FileID m_VirtualFileID;
 
-    // CI owns it
```

Comment:
One solution would be to also destroy the CI in a normal ROOT environment. That's why I started to touch the code in #794 which (once upon a time) actually called the dtor of our interpreter instance which in turn also destroys the DeclCollector. This will also avoid some nasty clang-related bugs in the future, as there actually is important code in those dtors that we currently don't execute at all (especially for modules)